src/index.ts

```diff
@@ -1,3 +1,52 @@
 import "./devices/devices-list";
 import "./components/esphome-header-menu";
 import "./components/esphome-fab";
+import "./editor/esphome-editor";
+import { LitElement, html } from "lit";
+import { customElement, state } from "lit/decorators.js";
+import { editCallback } from "./editor";
+
+@customElement("esphome-main")
+class ESPHomeMainView extends LitElement {
+  @state() private editing?: string;
+  constructor() {
+    super();
+    editCallback.callback = (configuration: string) => {
+      this.editing = configuration;
+    };
+  }
+
+  protected render() {
+    if (this.editing) {
+      return html`<style>
```

Comment:
We should use `static styles = css`

Reply:
again, see line 37: it does not work when skipping the shadowRoot with 
```
createRenderRoot() {
  return this;
}

Reply:
Ah that's correct.